src/Compilers/Core/Portable/Syntax/GreenNode.cs

```diff
@@ -370,6 +380,15 @@ public bool ContainsAnnotations
                 return (this.flags & NodeFlags.ContainsAnnotations) != 0;
             }
         }
+
+        public bool HasAnnotationsDirectly
```

Comment:
It feels reasonable to me to use this name throughout, for example, as the flag name as well.

Reply:
wfm

Reply:
don.e